docs/running_an_mpc_node_in_tdx_external_guide.md

```diff
@@ -456,9 +457,13 @@ For more information, see [local-key-provider-from-phala](https://github.com/Dst
 1. Follow the [canonical/tdx setup](#1-tdx-bare-metal-server-setup) if not already completed — especially step 9.1–2 (establishing an SGX PCCS: Provisioning Certification Caching Service).
 
 2. Deploy an instance of `gramine-sealing-key-provider` on the host machine.  
-   * On the DTX server, run the script [run.sh](https://github.com/Dstack-TEE/dstack/blob/master/key-provider-build/run.sh)  
+   * On the DTX server,  run the script [run.sh](https://github.com/Dstack-TEE/dstack/blob/master/key-provider-build/run.sh)  
```

Comment:
I think the is an extra space there that was added now

BTW in this line, is it DTX or TDX?